src/components/ContactPolicy/ContactPolicy.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Helmet } from 'react-helmet';
+import { useIntl } from 'react-intl';
+import HeaderSection from '../shared/HeaderSection/HeaderSection';
+import { InjectAppServices } from '../../services/pure-di';
+import { Breadcrumb, BreadcrumbItem } from '../shared/Breadcrumb/Breadcrumb';
+import { FormattedMessageMarkdown } from '../../i18n/FormattedMessageMarkdown';
+
+const ContactPolicy = () => {
+  const intl = useIntl();
+  const _ = (id, values) => intl.formatMessage({ id: id }, values);
+
+  return (
+    <>
+      <Helmet>
+        <title>{_('contact_policy.meta_title')}</title>
+      </Helmet>
+      <HeaderSection>
+        <div className="col-sm-12 col-md-12 col-lg-12">
+          <Breadcrumb>
+            <BreadcrumbItem href={_('common.control_panel_url')} text={_('common.control_panel')} />
+            <BreadcrumbItem text={_('contact_policy.title')} />
+          </Breadcrumb>
+          <h2>{_('contact_policy.title')}</h2>
+          {<FormattedMessageMarkdown id="contact_policy.subtitle" />}
+        </div>
+      </HeaderSection>
+      <section className="dp-container" />
+    </>
+  );
+};
+
+export default InjectAppServices(ContactPolicy);
```

Comment:
We are trying to avoid export defaults...

Reply:
Perhaps you could try this approach https://github.com/FromDoppler/doppler-webapp/blob/eb1c3ea03a97b997f705e0fd8bf72ee326a260d8/src/components/SiteTrackingRequired/SiteTrackingRequired.js#L15